web/src/admin.js

```diff
@@ -229,7 +229,6 @@ export function build_page() {
 }
 
 export function launch(section) {
-    settings.build_page();
     build_page();
```

Comment:
I think it was very much intentional that we called `build_page` for both settings and admin at the same time here. I think this means you should move both calls into `hashchange.js` if we're moving anything.